katka/signals.py

```diff
@@ -54,7 +54,7 @@ def send_pipeline_change_notification(sender, **kwargs):
         # being run, do not notify.
         return
 
-    session = settings.PIPELINE_CHANGE_NOTIFICATION_SESSION
+    session = settings.PIPELINE_RUNNER_SESSION
```

Comment:
I renamed this because this session is not only used for pipeline notification changes, but now also for step updates